.reaction/docker/scripts/build-meteor.sh

```diff
@@ -1,30 +1,35 @@
 #!/bin/bash
 
 #
-# builds a production meteor bundle directory
+# builds a production meteor bundle
 #
 set -e
 
-printf "\n[-] Building Meteor application...\n\n"
-
-: ${APP_BUNDLE_DIR:="/var/www"}
-: ${BUILD_SCRIPTS_DIR:="/opt/reaction"}
-
 cd $APP_SOURCE_DIR
 
 # Customize packages
-bash $BUILD_SCRIPTS_DIR/build-packages.sh
+$BUILD_SCRIPTS_DIR/build-packages.sh
 
 # Generate plugin import files
-bash $BUILD_SCRIPTS_DIR/plugin-loader.sh
+printf "\n[-] Running Reaction plugin loader...\n\n"
+$BUILD_SCRIPTS_DIR/plugin-loader.sh
 
 # Install app deps
+printf "\n[-] Running npm install...\n\n"
 meteor npm install
 
-# build the source
+# build the production bundle
+printf "\n[-] Building Meteor application...\n\n"
 mkdir -p $APP_BUNDLE_DIR
-meteor build --directory $APP_BUNDLE_DIR
-cd $APP_BUNDLE_DIR/bundle/programs/server/ && meteor npm install --production
+meteor build --unsafe-perm --directory $APP_BUNDLE_DIR
```

Comment:
this looks suspicious. since the assumption is that this is run in non root mode, why would you need ```--unsafe-perm```? unless i am missing something